user_manual/files/federated_cloud_sharing.rst

```diff
@@ -6,6 +6,9 @@ Federation Sharing allows you to mount file shares from remote ownCloud servers,
 creating your own cloud of ownClouds. You can create direct share links with 
 users on other ownCloud servers.
 
+.. important:: 
+   Sharing to federated groups is not supported.
```

Comment:
"federated groups" isn't even a thing. I'd say "Sharing to groups from federated ownCloud instances is not supported".